docs/databags.md

```diff
@@ -33,6 +33,15 @@ The License Hash is part of a data bag item encrypted with [Chef Vault](https://
 
 * `[A Decriptive File-Name]` - Corresponding License (XML) contents. (There can be many of these) Remember to change newlines to `\n` to conform to proper JSON format.
 
+License Pool Hash
+------------
+This hash is part of a plaintext data bag item that defines multiple license pools. Indexers can be assigned to a specific pool and license quota can be set for each pool by configuring this databag. 
+
+* `['total_pool_size']` - The total available pool size.
+* `['auto_generated_pool_size']` - This is the pool size for the auto generated pool. Indexers that tries to connect to the license server, when not assinged to a specific pool will land in this pool.
```

Comment:
minor, grammar. "Indexers that try to connect ..." or just reword to "Indexers that connect to the license server ..."

also spelling on assigned.